src/hangman.rs

```diff
@@ -0,0 +1,106 @@
+#[derive(Default)]
+pub struct Hangman {
+    word: String,
+    attempts: usize,
+    misses: [char; 5],
+    hits: Vec<char>,
+}
+
+pub enum AttemptResult {
+    CanKeepPlaying,
+    Win,
+    Hanged,
+}
+
+const SYMBOL_LETTER_TO_COMPLETE:char = '_';
+
+impl Hangman {
+    pub fn new(word: String) -> Hangman {
+        let vec_size = word.len();
+        Hangman { 
+            word, 
+            attempts: 0, 
+            misses: Default::default(), 
+            hits: vec![SYMBOL_LETTER_TO_COMPLETE; vec_size],
+        }
+    }
+
+    fn guess_letter(&mut self, l:char) -> AttemptResult {
```

Comment:
Please change the variable to a descriptive name.

Reply:
Changed l to letter